caikit/core/data_model/streams/data_stream.py

```diff
@@ -176,7 +176,8 @@ def generator_func():
 
                 try:
                     for line in lines:
-                        yield json.loads(line)
+                        if line != b"\n":  # ignore new line characters
```

Comment:
there has to be a better way to do this?

Reply:
Can you just do `if not line.strip(): continue`?

Reply:
Since we open the file in binary mode (I'm still not sure I understand why), `line.strip` gives me `b''` which is why I wasn't able to use that and I had to resort to checking `line != b"\n"` explicitly.